### PR TITLE
Ace 5842:csrf filter in docker

### PR DESCRIPTION
**Pull request by ciprianchelaru**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
    </parent>
    <groupId>org.alfresco.surf</groupId>
    <artifactId>spring-surf-webscripts-parent</artifactId>
-   <version>6.16-SNAPSHOT</version>
+   <version>6.18-SNAPSHOT</version>
```
Comment (killerboot) [inline pom.xml]: Why are these appearing here? Should you merge to master?
What is the purpose of develop in this project?

File: spring-webscripts/spring-webscripts/src/main/java/org/springframework/extensions/webscripts/servlet/CSRFFilter.java [hunk 2]
```diff
@@ -960,7 +962,17 @@ public void init(Map<String, String> params) throws ServletException
             }
 
             // Override the xml by alfresco-global.properties
-            Properties globalProperties = (Properties) getApplicationContext().getBean("global-properties");
+            Properties globalProperties = null;
+            try
+            {
+                globalProperties = (Properties) getApplicationContext().getBean("global-properties");
+            }catch (NoSuchBeanDefinitionException exc){
```
Comment (killerboot) [inline spring-webscripts/spring-webscripts/src/main/java/org/springframework/extensions/webscripts/servlet/CSRFFilter.java]: Please fix the formatting.

File: spring-webscripts/spring-webscripts/src/main/java/org/springframework/extensions/webscripts/servlet/CSRFFilter.java [hunk 3]
```diff
@@ -960,7 +962,17 @@ public void init(Map<String, String> params) throws ServletException
...
+                if (logger.isWarnEnabled())
```
Comment (killerboot) [inline spring-webscripts/spring-webscripts/src/main/java/org/springframework/extensions/webscripts/servlet/CSRFFilter.java]: I think it should be DEBUG as the a-g.p are missing everywhere except Alfresco web app.

File: spring-webscripts/spring-webscripts/src/main/java/org/springframework/extensions/webscripts/servlet/CSRFFilter.java [hunk 4]
```diff
@@ -1090,7 +1101,17 @@ public void init(Map<String, String> params) throws ServletException
             }
 
             // Override the xml by alfresco-global.properties
-            Properties globalProperties = (Properties) getApplicationContext().getBean("global-properties");
+            Properties globalProperties = null;
+            try
+            {
+                globalProperties = (Properties) getApplicationContext().getBean("global-properties");
+            }catch (NoSuchBeanDefinitionException exc){
```
Comment (killerboot) [inline spring-webscripts/spring-webscripts/src/main/java/org/springframework/extensions/webscripts/servlet/CSRFFilter.java]: Please fix formatting.

File: spring-webscripts/spring-webscripts/src/main/java/org/springframework/extensions/webscripts/servlet/CSRFFilter.java [hunk 5]
```diff
@@ -1090,7 +1101,17 @@ public void init(Map<String, String> params) throws ServletException
...
+                if (logger.isWarnEnabled())
```
Comment (killerboot) [inline spring-webscripts/spring-webscripts/src/main/java/org/springframework/extensions/webscripts/servlet/CSRFFilter.java]: See, previous. Maybe a DEBUG instead.

File: pom.xml (hunk 1 above)
Comment (ciprianchelaru) [reply pom.xml]: I saw that the release was made from master and the version is 6.17. So when I created this branch for fix (ACE-5842_CSRF_filterInDocker) there was an misalignment, master has version 6.18-SNAPSHOT and develop 6.16-SNAPSHOT . If we work on develop , I think that those should be aligned, right?

Comment (killerboot) [reply pom.xml]: I'm not sure if you need develop at all. In projects like https://github.com/Alfresco/alfresco-repository we don't use develop at all. I think it makes sense to remove develop here as well.

Comment (ciprianchelaru) [reply pom.xml]: When we made the Share Flow we inspired from the repo flow which had develop branch

**Pull request merged**